in_toto/gpg/__init__.py

```diff
@@ -0,0 +1,2 @@
+class PacketParsingError(Exception):
```

Comment:
Should these exception definitions go in `in_toto/exceptions.py` or `in-toto/gpg/exceptions.py`?